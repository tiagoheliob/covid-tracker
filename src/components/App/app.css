body {
    background: #ecf0f1 !important;
}