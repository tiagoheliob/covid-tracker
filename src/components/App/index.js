import React from 'react';
import AppBar from '../AppBar'
import Main from '../../pages/Main/';
import { Provider } from 'react-redux';

import store from '../../redux/store';
import './app.css';

export default () => {
    return (
        <Provider store={store}>
            <AppBar />
            <Main />
        </Provider>
    );
}