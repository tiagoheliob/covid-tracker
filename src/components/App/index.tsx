import { ThemeProvider } from "styled-components";
import Navbar from "../Navbar";
import { Provider } from "react-redux";
import { BrowserRouter, Route } from "react-router-dom";
import CountriesList from "../CountriesList/countries-list";
import { GlobalStyle, theme } from "../../styled-configuration/globalStyle";

import store from "../../redux/store";
import "./app.css";

export default () => {
  return (
    <Provider store={store}>
      <ThemeProvider theme={theme}>
        <Navbar />
        <GlobalStyle />
        <BrowserRouter forceRefresh={true}>
          <Route component={CountriesList} path="/" exact />
        </BrowserRouter>
      </ThemeProvider>
    </Provider>
  );
};
