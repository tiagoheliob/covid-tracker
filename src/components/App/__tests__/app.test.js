import React from 'react';
import { shallow } from 'enzyme';
import App from '../index';


let wrapped;

beforeEach(() => {
    wrapped = shallow(
        <App />
    );
});

describe('The login component', () => {
    
    it('Check if matches the snapshot', () => {
        expect(wrapped).toMatchSnapshot();
    });

});