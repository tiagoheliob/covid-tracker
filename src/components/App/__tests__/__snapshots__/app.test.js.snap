// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The login component Check if matches the snapshot 1`] = `ShallowWrapper {}`;
